tests/PySys/mqtt_port/mqtt_port_change_connection_fails/run.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import time
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+from pysys.basetest import BaseTest
+
+
+"""
+Validate changing the mqtt port using the tedge command fails without restarting the mqtt server
+
```

Comment:
I don't really understand the scope of this test. I suppose that mqtt server is mosquitto. Does it fail as tedge is using the old port and mosquitto the old one?

Reply:
I added title same as the test folder name